test/helpers/core.js

```diff
@@ -11,6 +11,5 @@ export default function createCore() {
     relayServiceEndpoint: process.env.RELAY_SERVICE_ENDPOINT,
     safeMasterAddress: process.env.SAFE_ADDRESS,
     subgraphName: process.env.SUBGRAPH_NAME,
-    databaseSource: process.env.DATABASE_SOURCE,
```

Comment:
without this vr in the tests?

Reply:
exactly, because I added a default value, and therefore is not mandatory

Reply:
test pass even in the GitHub Actions